operators/multiclusterobservability/controllers/multiclusterobservability/observatorium.go

```diff
@@ -926,9 +933,10 @@ func newCompactSpec(mco *mcov1beta2.MultiClusterObservability, scSelected string
 	compactSpec := obsv1alpha1.CompactSpec{}
 	// Compactor, generally, does not need to be highly available.
 	// Compactions are needed from time to time, only when new blocks appear.
-	compactSpec.Replicas = &mcoconfig.Replicas1
+	var replicas1 int32 = 1
+	compactSpec.Replicas = &replicas1
```

Comment:
should we even be exposing replicas on the spec of compactor? it should run as a singleton?

Reply:
This is hardcoded, I'm not sure why the spec has replicas field